hardhat.config.js

```diff
@@ -1,6 +1,17 @@
 require('@nomicfoundation/hardhat-toolbox');
 require('solidity-coverage');
 
+const fs = require('fs');
+const env = process.env.ENV || 'testnet';
+const {
+  importNetworks,
+} = require('@axelar-network/axelar-contract-deployments/evm/utils');
+const chains = require(`@axelar-network/axelar-contract-deployments/info/${env}.json`);
+const keys = fs.existsSync(`${__dirname}/info/keys.json`)
```

Comment:
can make it a bit cleaner
https://github.com/axelarnetwork/axelar-cgp-solidity/blob/main/hardhat.config.js#L3

Reply:
fixed